packages/core-mobile/app/services/notifications/NotificationsService.ts

```diff
@@ -57,7 +54,10 @@ class NotificationsService {
    * Tries to pull up system prompt for allowing notifications, if that doesn't
    * work opens system settings
    */
-  async getAllPermissions(shouldOpenSettings = true) {
+  async getAllPermissions(shouldOpenSettings = true): Promise<{
+    permission: string
```

Comment:
nit: use enum or something else instead of string. no need to do it in this pr.

Reply:
Since only 2 values i think string literals are good enough
[Catch errors](https://github.com/ava-labs/core-mobile/pull/1659/commits/d39fc40926831a5f09830a5a56ad4a45e3ef3240)